e2e/assets/single-cluster/helm-options-skip-schema-validation.yaml

```
@@ -0,0 +1,10 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: helm-options-skip-schema-val # truncate name to control bundle name length
```

nit: looks like this comment is not needed. Can you remove it?
```suggestion
  name: helm-options-skip-schema-val 
```

---

I put a comment because the name does not follow any existing convention. I can remove it